src/indigo/less/utilities/spacing.less

```diff
@@ -40,14 +32,6 @@
   .pl-@{n} {
     padding-left: @n * @base !important;
   }
-  .px-@{n} {
-    padding-left: @n * @base !important;
-    padding-right: @n * @base !important;
-  }
-  .py-@{n} {
-    padding-top: @n * @base !important;
-    padding-bottom: @n * @base !important;
-  }
```

Comment:
Všiml jsme si že generuje i tyto helpery, ale v kbc-ui není ani jedno použití tak to mažu.. asi nám stačí ty základní v novém design systému to třeba i tak vymyslíme ještě jinak.